dtfs-central-api/test-helpers/test-data/eligibility-criteria-amendments.ts

```diff
@@ -0,0 +1,17 @@
+import { ObjectId } from 'mongodb';
+import { AmendmentsEligibilityCriteria, DEAL_TYPE, FACILITY_TYPE } from '@ukef/dtfs2-common';
+import { generateMockPortalUserAuditDatabaseRecord } from '@ukef/dtfs2-common/change-stream/test-helpers';
+
+export const anAmendmentsEligibilityCriteria = (): AmendmentsEligibilityCriteria => ({
+  _id: new ObjectId(),
+  version: 1,
+  product: DEAL_TYPE.GEF,
```

Comment:
I think we can safely remove `product` attribute provided we have `facilityType` for conditional assertion.